test/d2l-applied-filters/d2l-applied-filters.html

```diff
@@ -101,6 +101,107 @@
 						done();
 					});
 				});
+
+				test('clear filters button is hidden when 3 filters are applied', function() {
```

Comment:
There is html at the top of this file that makes it so 3 filters are selected when the page first renders